prac_05/word_occurrences.py

```diff
@@ -0,0 +1,32 @@
+"""
+Word Occurrences
+Estimate: 40 minutes
+Actual:   61 minutes
+"""
+
+
+def main():
+    text_to_number = {}
+    text = input("Text: ")
+    texts = text.split(" ")
```

Comment:
Probably use something that we always use when we do split(). Like parts?

Reply:
parts is not applicable for this event.